package config;

message TransfersvrConfig {
	optional string listen_ip = 1;
	optional int32 svr_inst_id = 2;
	optional int32 svr_zone = 3;
}